File: Calculator.py
import sys

sys.ps1 = '\033[01;32m '
print(sys.ps1)

def multiply():
 a1 = float(input('Enter the number 1st number: '))
 b1 = float(input('Enter the 2nd number: '))
 print(a1*b1)

def plus():
 a1 = float(input('Enter the number 1st number: '))
 b1 = float(input('Enter the 2nd number: '))
 print(a1+b1)
 

def minus():
 a1 = float(input('Enter the number 1st number: '))
 b1 = float(input('Enter the 2nd number: '))
 print(a1-b1)
 
 

def divide():
 a1 = float(input('Enter the number 1st number: '))
 b1 = float(input('Enter the 2nd number: '))
 print(a1/b1)
 
l = input('Do ya wanna multiply, plus, divide or minus? [OPTIONs:minus/plus/divide/multiply]: ')

if (l == "plus"):
 plus()
elif (l == "minus"):
 minus()
elif (l == "divide"):
 divide()
elif (l == "multiply"):
 multiply()
else:
 print("Sorry but there\'s no option for that. pleae make sure you spelled correctly. the choices are : minus, plus, divide of multiply")

*You can use it if want*

File: calculator.cpp
#include <iostream>

using namespace std;
void plus(){
    
    
       float a;
       cout << "Enter first value : "<<endl;
       cin>>a;

    
       float b;
       cout << "Enter second value : "<<endl;
        cin>>b;
        cout<<a+b;
		
}
void minus(){
    
    
    float a;
    cout << "Enter first value : "<<endl;
    cin>>a;

    
    float b;
    cout << "Enter second value : "<<endl;
    cin>>b;

    
    cout<<a-b;
}

void divide(){
    
    
    float a;
    cout << "Enter first value : "<<endl;
    cin>>a;

    
    float b;
    cout << "Enter second value : "<<endl;
    cin>>b;

    
    cout<<a/b;
}

void multiply(){
    
    
    float a;
    cout << "Enter first value : "<<endl;
    cin>>a;

    
    float b;
    cout << "Enter second value : "<<endl;
    cin>>b;

    
    cout<<a*b;
}

int main()
{
	



    cout << "Hello World!" << endl;
    
    string c;
    cout << "choose an option, options are plus / minus / divide / multiply : ";
    
    cin >> c;
    
    if (c == "plus") {
      void plus();
       plus();

    }
    else if (c == "minus") {
      void minus();
      minus();
        
    }
    
   else if (c == "multiply") {
     
     void multiply();
     multiply();
     
      } 
   else if (c == "divide") {
     
     void divide();
     divide();
     
     }
	else {
		cout << "No valid option"<<endl;
	}
    

    return 0;
}

